packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -559,9 +551,7 @@ export interface ISharedString extends ISharedSegmentSequence<SharedStringSegmen
 }
 
 // @public
-export interface ISubscribable<E extends Events<E>> {
-    on<K extends keyof Events<E>>(eventName: K, listener: E[K]): Off;
-}
+export type IsListener<TListener> = TListener extends (...args: any[]) => unknown ? true : false;
```

Comment:
I see this return value changed from any to unknown which seems fine, but should it be void?

Reply:
Good idea, `void` is better. I think it typechecks the same as `unknown`. `unknown` would be more appropriate if we were keeping `emitAndCollect` around, but since we don't want to, I think `void` here is the way to go.